src/index.js

```diff
@@ -90,17 +93,20 @@ function createSolutionInfos() {
   ));
 }
 
-function processNext(next, config) {
-  const branch = config.submodules[next];
-  console.log(`Processing ${next}`);
+function processSubmodule(name, config) {
+  const branch = config.submodules[name];
 
-  const [owner, repo] = next.split('/');
+  const [owner, repo] = name.split('/');
   const submodule = {
+    _logs: [],
     owner,
     repo,
     branch,
-    remoteUrl: `git@git:${owner}/${repo}.git`
+    remoteUrl: `git@git:${owner}/${repo}.git`,
+    log(message, indent=0) { this._logs = this._logs.concat(message.split(/\r?\n/).map((line) => { return Array(indent + 1).join(' ') + line; })); },
```

Comment:
`clone-leeroy` *is* ES6. If you need an ES6 repl, use `babel-node`.

Sorry I missed the point of the + 1 in the old code. `' '.repeat(indent) + line` should work instead, right?